StRoot/StETofMatchMaker/StETofMatchMaker.h

```diff
@@ -98,6 +99,11 @@ class StETofMatchMaker : public StMaker {
         Double_t        beta;
         Double_t        pathLength;
         Double_t        tof;
+
+        Int_t           IdTruth;     // merge
+        Int_t           IdTruthHit;  // merge
+	Int_t           nHitsFit;    // merge
+	bool            isMissMatch; // merge
```

Comment:
```suggestion
        Int_t           idTruth;
        Int_t           idTruthHit;
	Int_t           nHitsFit;
	bool            isMissMatch;
```

See https://star-bnl.github.io/star-coding-guidelines/formatguide.xml#Variable_Names for the variable naming convention